lib/cadence/saga/compensable.rb

```diff
@@ -0,0 +1,23 @@
+
+module Cadence
+  module Saga
+    module Compensable
+      def compensable_errors(*args)
+        return @compensable_errors if args.empty?
+        @compensable_errors = args.first
+      end
+
+      def non_compensable_errors(*args)
```

Comment:
What do you think about moving this as arguments to the `run_saga` method instead?

I think it makes a bit more sense this way and allow for things like 2 independent `run_saga` blocks in the same workflow, etc. Coupling it to the workflow itself feel unnecessary and potentially restrictive

Reply:
Makes sense to me. I was thinking about ways to make it more configurable. I added an optional config hash that can be passed to `run_saga`